src/containers/App/saga.test.ts

```diff
@@ -167,7 +171,8 @@ describe('containers/App/saga', () => {
     });
 
     afterEach(() => {
-      global.location.reload.mockRestore();
+      // eslint-disable-next-line no-extra-parens
```

Comment:
These parens don't seem to be 'extra'. Could we tweak the linting rules, so we don't need to disable this rule in such cases?

Reply:
Agree, is just for now. I will ask @jpoppe to help with this.